dipy/tracking/streamline.py

```diff
@@ -251,3 +251,63 @@ def select_by_rois(streamlines, rois, include, mode=None, affine=None,
                                       mode=mode)
         if include & ~exclude:
             yield sl
+
+
+def orient_by_rois(streamlines, roi1, roi2, affine=None):
+    """Orient a set of streamlines according to a pair of ROIs
+
+    Parameters
+    ----------
+    streamlines : list
+        List of 3d arrays. Coordinates of the
+    roi1, roi2 : ndarray
+        Binary masks designating the location of the regions of interest, or
+        coordinate arrays (3d coordinates)
+    affine : ndarray
+        Affine transformation from voxels to streamlines. Default: identity.
```

Comment:
Since this is something that pops up in almost all of our internal discussion on streamlines: should the affine documentation specify if the offset should be included in the transformation? By offset, I mean the same offset that is used when mapping streamlines to voxel coordinates to interact with a mask, which is normally 0.5 * the voxel size.

I would say that it is up to the user, and in most cases, it probably won't change the end result, but I'm curious what others would say.